samples/betaFeatures.js

```diff
@@ -0,0 +1,547 @@
+  const speech = require('@google-cloud/speech').v1p1beta1;
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const fileName = 'Local path to audio file, e.g. /path/to/audio.raw';
+
+  const config = {
+    encoding: `FLAC`,
+    sampleRateHertz: 16000,
+    languageCode: `en-US`,
+    enableWordConfidence: true,
+  };
+
+  const audio = {
+    content: fs.readFileSync(fileName).toString('base64'),
+  };
+
+  const request = {
+    config: config,
+    audio: audio,
+  };
+
+  client
+    .recognize(request)
+    .then(data => {
+      const response = data[0];
+      const transcription = response.results
+        .map(result => result.alternatives[0].transcript)
+        .join('\n');
+      const confidence = response.results
+        .map(result => result.alternatives[0].confidence)
+        .join(`\n`);
+      console.log(
+        `Transcription: ${transcription} \n Confidence: ${confidence}`
+      );
+
+      console.log(`Word-Level-Confidence:`);
+      const words = response.results.map(result => result.alternatives[0]);
+      words[0].words.forEach(a => {
+        console.log(` word: ${a.word}, confidence: ${a.confidence}`);
+      });
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END speech_transcribe_word_level_confidence]
+}
+
+function speechTranscribeWordLevelConfidenceGCS(gcsUri) {
+  // [START speech_transcribe_word_level_confidence_gcs]
+
+  // Imports the Google Cloud client library
+  const speech = require('@google-cloud/speech').v1p1beta1;
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const uri = path to GCS audio file e.g. `gs:/bucket/audio.wav`;
+
+  const config = {
+    encoding: `FLAC`,
+    sampleRateHertz: 16000,
+    languageCode: `en-US`,
+    enableWordConfidence: true,
+  };
+
+  const audio = {
+    uri: gcsUri,
+  };
+
+  const request = {
+    config: config,
+    audio: audio,
+  };
+
+  client
+    .recognize(request)
+    .then(data => {
+      const response = data[0];
+      const transcription = response.results
+        .map(result => result.alternatives[0].transcript)
+        .join('\n');
+      const confidence = response.results
+        .map(result => result.alternatives[0].confidence)
+        .join(`\n`);
+      console.log(
+        `Transcription: ${transcription} \n Confidence: ${confidence}`
+      );
+
+      console.log(`Word-Level-Confidence:`);
+      const words = response.results.map(result => result.alternatives[0]);
+      words[0].words.forEach(a => {
+        console.log(` word: ${a.word}, confidence: ${a.confidence}`);
+      });
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END speech_transcribe_word_level_confidence_gcs]
+}
+
+//audio file paths for Defaults
```

Comment:
Can you point to a precedent for this, i.e. another sample that does this?

I've never seen this before, where did you copy this from?

This isn't necessary, we do not provide default values for localfile arguments which the user passes to the CLI

If there is no precedent, remove this code & remove the defaults from the options

I do not want to maintain this code

Reply:
I cleaned this up, users will enter file name on yaargs CLI interface - idea was that it would save time, but wasn't being used elsewhere, sorry about that